boofuzz/sessions.py

```diff
@@ -373,13 +373,27 @@ def __init__(self, session_filename=None, index_start=1, index_end=None, sleep_t
         self._receive_data_after_fuzz = receive_data_after_fuzz
         self._skip_current_node_after_current_test_case = False
         self._skip_current_element_after_current_test_case = False
-        self._pre_send_methods = pre_send_callback
-        self._post_test_case_methods = post_test_case_callback
-        self._restart_methods = restart_callback
+
+        self._post_test_case_methods = []
```

Comment:
looks like this bit is redundant with lines 387-390.

Reply:
Thanks for the hint, that one must have sneaked in when I merged the upstream changes.